### PR TITLE
Add Syphon to client list

### LINKED ISSUE #795: Add Syphon to matrix.org/clients
See https://github.com/vector-im/riot-web/issues/14800

### PR DESCRIPTION
**Pull request by TR-SLimey**
Fixes #795

### CODE REVIEW

File: gatsby/content/projects/clients/syphon.mdx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    Display formatted messages: no
+    Redacting: no
+    Multiple Matrix Accounts: no
+    New user registration: yes
+    voip: no
+    Reactions: yes
+    Message editing: Partial
+    Room upgrades: no
+    Localisations: 2
+    SSO Support: no
```
Comment (ereio) [inline gatsby/content/projects/clients/syphon.mdx:53]: Syphon does support SSO, only one I've tested extensively however is mozilla's homeserver

Comment (TR-SLimey) [reply gatsby/content/projects/clients/syphon.mdx]: Ah, thanks for pointing that out. Fixed :D

File: gatsby/content/projects/clients/syphon.mdx [hunk 2]
```diff
@@ -0,0 +1,62 @@
+---
+layout: projectimage
+title: Syphon
+categories:
+ - client
+description: a not for profit, open source matrix client with a focus on privacy and ease of use
```
Comment (ereio) [inline gatsby/content/projects/clients/syphon.mdx:6]: Upper case the A in "A not for profit..." and we should be good!

Comment (TR-SLimey) [reply gatsby/content/projects/clients/syphon.mdx]: Done. Marking as ready :D

**Comment by ereio** [comment]
<details>
  <summary>Small Preview</summary>
  Screen Shot 2021-03-05 at 4 00 24 PM
</details>
 <details>
  <summary>Full Preview</summary>
Screen Shot 2021-03-05 at 3 58 21 PM
</details>

**Review by benparsons — APPROVED** [review]
lgtm

**Pull request merged**